packages/compiler/src/formatter/print/printer.ts

```diff
@@ -1132,6 +1180,44 @@ export function printModelProperty(
   ];
 }
 
+export function printModelValidate(
+  path: AstPath<ModelValidateNode>,
+  options: TypeSpecPrettierOptions,
+  print: PrettierChildPrint
+) {
+  const node = path.getValue();
+  const propertyIndex = path.stack[path.stack.length - 2];
+  const isNotFirst = typeof propertyIndex === "number" && propertyIndex > 0;
+  const { decorators, multiline } = printDecorators(
+    path as AstPath<DecorableNode>,
+    options,
+    print,
+    {
+      tryInline: true,
+    }
+  );
+  if (node.id === undefined) {
+    return [
+      multiline && isNotFirst ? hardline : "",
+      printDirectives(path, options, print),
+      decorators,
+      "validate ",
+      path.call(print, "value"),
+    ];
+  } else {
+    const id = printIdentifier(node.id, options);
```

Comment:
```suggestion
    const id = node.id ? [printIdentifier(node.id, options), ": "] : "";
```
and change below to just pass id and you can remove the if else